cookiecutter/{{cookiecutter.organization}}-richie-site-factory/template/{{cookiecutter.site}}/src/frontend/scss/extras/settings/colors/_richie_upstream_variables.scss

```diff
@@ -0,0 +1,170 @@
+// File with the variables defined in:
```

Comment:
Hmmm... I wonder if we should not suffix with `!default` all these variables from `https://github.com/openfun/richie/blob/master/src/frontend/scss/settings/_variables.scss` then lets developer override through this file only variables it wants like we are doing for scheme.

Reply:
+1 or it will be hard to maintain?

Reply:
I think we can't add `!default` because the current definition is to override the default bootstrap values.
I remember that when I tried to refactor the theme code during https://github.com/openfun/richie/pull/1540. 

In my opinion we should have a scss file in Richie with the content of the `_richie_upstream_variables.scss`.
So the `_variables.scss` can be more compact and we can reuse the file on the `richie-site-factory` repos.

Other improvement that I think we should implement is to create a more compact version of the `_main.scss`. Further upgrades of richie-site-factories would be simpler and the code required for a lighter theme customization would be more compact.

Reply:
Yes @igobranco, I think it is good idea!
It is currently a painpoint of richie-site-factory to maintain de `_main.scss` stylesheet on each upgrade.

Reply:
Done.
I'm going to create a new PR so we can use it has a discussion to refactor the scss code. After the conference we can use it